static/app/actionCreators/organization.tsx

```diff
@@ -42,7 +42,11 @@ async function fetchOrg(
   }
 
   FeatureFlagOverrides.singleton().loadOrg(org);
-  FeatureObserver.singleton().observeFlags({organization: org, bufferSize: 100});
+  FeatureObserver.singleton({
+    bufferSize: FEATURE_FLAG_BUFFER_SIZE,
+  }).observeOrganizationFlags({
+    organization: org,
+  });
```

Comment:
could skip passing in the `bufferSize` value here, the default value will be fine to rely on. it already optional
```suggestion
  FeatureObserver.singleton({}).observeOrganizationFlags({organization: org});
```